upcloud/network.go

```diff
@@ -292,6 +292,14 @@ type Router struct {
 	Type             string             `json:"type"`
 	UUID             string             `json:"uuid"`
 	Labels           []Label            `json:"labels"`
+	StaticRoutes     []StaticRoute      `json:"static_routes"`
+}
+
+// StaticRoute represents a Static route
+type StaticRoute struct {
+	Name    string `json:"name"`
+	Route   string `json:"route"`
+	Nexthop string `json:"nexthop"`
```

Comment:
Should we use NextHop ? :thinking:

Reply:
I opted to use `Nexthop` for the field (`nexthop`) conversion, as it the original does not contain underscore. I do agree that `NextHop` would "look" better, but I would still keep it as it is to follow previous conventions.